library/cartridge_get_shortened_fact_sets.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+
+from ansible.module_utils.helpers import Helpers as helpers
+
+argument_spec = {
+    'hostvars': {'required': True, 'type': 'dict'},
+}
+
+sets_list = {
```

Comment:
I don't like the naming.

It's `facts_by_steps`, `facts_by_targets` or smth like this.

Reply:
Good comment, I missed some of the variables. Added a test for this.